src/modules/file/file.repository.ts

```diff
@@ -205,6 +109,24 @@ export class SequelizeFileRepository implements FileRepository {
       limit,
       offset,
       where,
+      include: [
```

Comment:
This should be optional, we do not want to do queries that make things slower without any reason

Reply:
With the parameter 'required: false' below, isn't it optional?

Reply:
Required does [mean](https://sequelize.org/docs/v6/advanced-association-concepts/eager-loading/#required-eager-loading) that the query is an INNER or an OUTER JOIN, but the database, in both cases, is trying to get the results. The difference between them is that if is true, then the file should be returned only if the included models have a row related to the file (INNER) otherwise, the file will be always returned (OUTER). 

We are looking to be able to do a `required: false` of thumbnails if and only if we need it. Therefore, I would recommend adding a new method called `findAllCursorWithThumbnails` which does the job, a controller query param to request the inclusion, and an if on the use case that calls findAllCursor or findAllCursorWithThumbnails depending on that.